lib/grape/endpoint.rb

```diff
@@ -165,10 +165,17 @@ def prepare_routes
     end
 
     def prepare_path(path)
-      path_settings = inheritable_setting.to_hash[:namespace_stackable].merge(inheritable_setting.to_hash[:namespace_inheritable])
       Path.prepare(path, namespace, path_settings)
     end
 
+    def path_settings
+      inheritable_setting.to_hash[:namespace_stackable].merge(inheritable_setting.to_hash[:namespace_inheritable])
+    end
+
+    def uses_specific_format?
```

Comment:
This method is identical to `Path#uses_specific_format?`. There’s probably a better way to structure things so that this duplication is not necessary.